llvm/test/CodeGen/X86/peephole-copy.ll

```diff
@@ -0,0 +1,20 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 3
+; RUN: llc < %s -mtriple=i686-- | FileCheck %s
```

Comment:
Could you make this only a peephole test using `.mir`?

The description mentions a `COPY` but from the IR it is unclear where it comes from and what happens to it in the peephole optimizer.

Reply:
done.